forge-gui/src/main/java/forge/player/PlayerControllerHuman.java

```diff
@@ -523,12 +523,8 @@ private CardCollectionView choosePermanentsTo(final int min, int max, final Card
             return CardCollection.EMPTY;
         }
 
-        String inpMessage = null;
-        if (min == 0) {
-            inpMessage = localizer.getMessage("lblSelectUpToNumTargetToAction", message, action);
-        } else {
-            inpMessage = localizer.getMessage("lblSelectNumTargetToAction", message, action);
-        }
+        String inpMessage = localizer.getMessage((min == 0 ? "lblSelectUpToNumTargetToAction" :
```

Comment:
ah that explains why it was the way it was then
thanks for breaking that down
the main thing I wanted to do was tidy up pluralization... to let Lang do it instead of {0}(s)... is that feasible?

Reply:
you'd have to pull the message String deeper into `InputSelectEntitiesFromList.getMessage` or preferably the Cards child instead

Reply:
Sounds good, will leave it alone for now